src/profile-logic/transforms.js

```diff
@@ -1128,8 +1128,7 @@ export function focusSubtree(
         if (stackMatchesUpTo === prefixDepth) {
           const newStackIndex = newStackTable.length++;
           const newStackPrefix = oldStackToNewStack.get(prefix);
-          newStackTable.prefix[newStackIndex] =
-            newStackPrefix !== undefined ? newStackPrefix : null;
+          newStackTable.prefix[newStackIndex] = newStackPrefix ?? null;
```

Comment:
I managed to find how this was transpiled to, after minification:
```
t=l.get(u);c.prefix[e]=null!=t?t:null
```
So `loose` seems to work as expected.